packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -220,6 +232,59 @@ const Breakouts = WebexPlugin.extend({
 
     this.breakouts.set(Object.values(breakouts));
   },
+
+  /**
+   * Make enable breakout resource
+   * @returns {Promise}
+   */
+  touchBreakout() {
```

Comment:
can you explain why the name "touchBreakout" for this function? I don't understand it, what does this http POST actually do? does it create a breakout session? in that case the name maybe should be "createBreakoutSession()".

Reply:
Before put create need check check active locus_dto.  maybe i can changed to checkLocusDTO?
https://sqbu-github.cisco.com/pages/WebExSquared/breakout/api/resource/enableBreakouts.html

Reply:
maybe I'm misunderstanding something, but looking at that locus wiki I think that POST just enables breakouts, so the best name for the function would be "enableBreakouts"

Reply:
Based on the document above, my understanding is that `POST` is used to enable breakouts for the provided Locus URL. Subsequently, we can use `PUT` to disable or re-enable breakouts.

Reply:
I try it again. Use POST to get Locus breakout URL and enable always. So the first time, there is no ned to call PUT to /breakout/api/v1/breakout/<uuid>